manim/mobject/graphing/scale.py

```diff
@@ -182,7 +182,7 @@ def get_custom_labels(
         tex_labels = [
             Integer(
                 self.base,
-                unit="^{%s}" % (f"{self.inverse_function(i):.{unit_decimal_places}f}"),
+                unit="^{%s}" % (f"{self.inverse_function(i):.{unit_decimal_places}f}"),  # noqa: UP031
```

Comment:
```suggestion
                unit="^{:.{prec}f}".format(self.inverse_function(i), prec=unit_decimal_places)
```

Reply:
This is kind of hard to understand, and also causes the tests to fail.

Reply:
I would say the original version is even more confusing. It was probably failing because of the missing `f"` prefix